src/components/JobConfiguration.vue

```diff
@@ -70,13 +70,40 @@
       <ion-button expand="block" @click="saveChanges()">{{ $t("Save changes") }}</ion-button>
     </div>
   </section>
+  <ion-list class="row">
```

Comment:
I would suggest to not use 2 separate lists to keep the items, instead of this please keep all the items in single div and apply flex-wrap property to it and also set some flex basis to the items.

Reply:
Updated ma'am